packages/codemod/src/plugin-deprecate/isBraidImport.ts

```diff
@@ -0,0 +1,4 @@
+export const isBraidImport = (importSource: string) =>
+  /(braid-design-system(?:\/css)?|(braid-src\/lib(?:\/(css|components))?))$/.test(
```

Comment:
Given that this regular expression will be executed a few hundred times, maybe it's worth extracting it to a constant in the module scope. What do you think?

Reply:
🏅 I'd split it into 2 regular expressions, for readability. Something like

```js
regexForBraid.test(importSource) || regexForInternal.text(importSource)
```

For most cases (consumers), the first branch is the one that will return early.